tests/interface/conftest.py

```diff
@@ -155,3 +155,31 @@ def s3_tester(interface_tester: InterfaceTester):
                         ),
                     )
                     yield interface_tester
+
+
+@pytest.fixture
+def grafana_datasource_tester(interface_tester: InterfaceTester):
+    with patch("lightkube.core.client.GenericSyncClient"):
+        with patch.multiple(
+            "cosl.coordinated_workers.worker.KubernetesComputeResourcesPatch",
+            _namespace="test-namespace",
+            _patch=lambda _: None,
+            is_ready=k8s_resource_patch_ready,
+        ):
+            with patch.multiple(
```

Comment:
can't this be combined with the above patch?

Reply:
Ok I'm blind

Reply:
copy-pastae'd directly from the scenario tests folder. 
Will give it a shot, but maybe there's a reason they're separate

Reply:
One was in `worker`, one in `coordinator` so let's see if it still works after the removal of one :)

Reply:
No I think its one for the worker and one for the coordinator